app/src/main/java/com/github/se/cyrcle/ui/zone/ZoneManagerScreen.kt

```diff
@@ -51,3 +104,57 @@ fun AddZoneButton(navigationActions: NavigationActions) {
         testTag = "AddButton")
   }
 }
+// Display relevant information about a zone. Allows the user to refresh the zone or delete it.
+@Composable
+fun ZoneCard(zone: Zone, zones: MutableState<List<Zone>>) {
```

Comment:
I guess that passing `zones` is ok, but wouldn't it be better to make it a `onDelete` callback, this might be more versatile?

Reply:
I don't like callback, I think they make the code harder to read 🙈 , especially here we don't need versatility as we would never call the `ZoneCard` with a different onDelete function

Reply:
Honestly, I can understand... (specially when we look at the ViewModels xD)

Reply:
Honestly I also think it's a bit weird to pass the whole List to a card, but if you think that this way is better, it's ok